### DISCUSSION TITLE
Charge until specific charge percentage

### DISCUSSION THREAD

**Discussion by FrankBakkerNl** [syssi/homeassistant-goecharger-mqtt#124] [Q&A]
Do any of the sensors provide information regarding the current charge level or expected range of the car? I would like to charge it until a minimum of say 50% or 150km range, regardless if solar power is available, and only charge further when I have surplus power.

**Comment by syssi** [comment]
This component cannot provide such information because there is no communication with your car. You should try to retrieve this information from your car instead.
